keycloak-radius-spi/.gitignore

```diff
@@ -0,0 +1,16 @@
+<<<<<<< Updated upstream
```

Comment:
This looks like a marge artifact that needs to be removed

Reply:
This is fixed now.